public/docs/installation/best-practice/allow-egress-kubernetes-api.yaml

```diff
@@ -0,0 +1,15 @@
+# allow-egress-kubernetes-api.yaml
+apiVersion: projectcalico.org/v3
```

Comment:
What would a policy for cilium look like? What is the more popular CNI?

Reply:
This might be interesting: https://github.com/awslabs/k8s-network-policy-migrator

Reply:
I'm dropping the Calico examples for now. There will only be an overview of network requirements and I will  leave it to the reader to figure out the configuration for their chosen CNI.